src/Paket.Core/PackageMetaData.fs

```diff
@@ -167,17 +167,39 @@ let findDependencies (dependenciesFile : DependenciesFile) config platform (temp
                     
                 deps, p :: files) ([], [])
     
-    // Add the assembly + pdb + dll from this project
+    // Add the assembly + {.dll, .pdb, .xml, /*/.resources.dll} from this project
     let templateWithOutput =
+        let projects =                 
+            if includeReferencedProjects then 
+                project.GetAllInterProjectDependenciesWithoutProjectTemplates() 
+                |> Seq.toList 
+            else 
+                [ project ]
+
+        let satelliteDlls =
+            seq {
+            for project in projects do
```

Comment:
(minor/style) not sure of general convention, but CE block should be indented (or put `seq {` at end of previous line)